install_config/router/default_haproxy_router.adoc

```diff
@@ -648,6 +648,88 @@ $ oc scale dc/router --replicas=0 && oc scale dc/router --replicas=1
 ----
 
 
+[[finding-router-hostname]]
+== Finding the Hostname of the Router
+
+When the user exposes a service, the host name in the route is the DNS name that external
+users will use to access the application. The network admin for the external network must
+make sure the host name resolves to the name of a router that has admitted the route.
+The user can set up their DNS with a CNAME that points to this hostname.
```

Comment:
We need to either say 'host name' or 'hostname' consistently throughout.  I think hostname is now preferred in the later RFCs.